### PR TITLE
Add usage examples to README

### PR DESCRIPTION
**Pull request by louis-ver**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,58 +3,137 @@
...
-> # In a Windows environment
-> set PIPELINE_DIRECTORY=C:\path\to\pipelines
-> set DATA_DIRECTORY=C:\path\to\data
+This request returns a `json` object containing the `apiKey` that will be required
+for authenticated queries.
+
+```json
+{
+  "httpHeader": "apiKey",
+  "httpHeaderValue": "[super-secret-api-key]"
```
Comment (simon-dube) [inline README.md]: I think "user-secret-api-key" would be enough.

File: README.md [hunk 2]
```diff
@@ -3,58 +3,137 @@
...
+     -H 'apiKey: [super_secret_api_key]'
 ```
 
-## Usage
+The server should return a `Path` object, which describes the resource that we uploaded:
+
+```json
+{
+  "platformPath": "http://localhost:8080/path/admin/hello_world.txt",
+  "lastModificationDate": 1521740108,
```
Comment (simon-dube) [inline README.md:130]: Please add a comment indicating tha the "lactModificationDate" should be different, just for safe measure.

**Review by simon-dube — APPROVED** [review]
Please have a look at my two comments. Other than that, clear and simple description. Good job!

File: README.md (hunk 2 above)
Comment (louis-ver) [reply README.md:130]: As discussed, this will not be necessary.

**Pull request merged**